src/status_im/new_group/screen_private.cljs

```diff
@@ -64,11 +65,58 @@
            :font  :medium}
      (label :t/members-title)]
     #_[touchable-highlight {:on-press (fn [])}
-     [view st/add-container
-      [icon :add_gray st/add-icon]
-      [text {:style st/add-text} (label :t/add-members)]]]
+       [view st/add-container
+        [icon :add_gray st/add-icon]
+        [text {:style st/add-text} (label :t/add-members)]]]
     [list-view
      {:dataSource (to-datasource contacts)
       :renderRow  (fn [row _ _]
                     (list-item [new-group-contact row]))
       :style      st/contacts-list}]]])
+
+(defview new-contacts-group-toolbar [edit?]
+  [view
+   [status-bar]
+   [toolbar
+    {:title (label (if edit? :t/edit-group :t/new-group))}]])
+
+;;TODO: should be refactored into one common function for group chats and contact groups
+(defview contact-group []
```

Comment:
`contact-group` looks a bit hairy, will be great to split it into a few smaller functions